python/pyspark/ml/feature.py

```diff
@@ -167,6 +168,134 @@ def getSplits(self):
 
 
 @inherit_doc
+class CountVectorizer(JavaEstimator, HasInputCol, HasOutputCol):
+    """
+    Extracts a vocabulary from document collections and generates a [[CountVectorizerModel]],
+    which converts text documents to sparse vectors of token counts.
+
+    >>> df = sentenceData = sqlContext.createDataFrame([(0, ["a", "b", "c"]),
+    ... (1, ["a", "b", "b", "c", "a"])], ["label", "raw"])
```

Comment:
The following style might be better:

~~~python
df = sqlContext.createDataFrame(
    [(0, ["a", "b", "c"]), (1, ["a", "b", "b", "c", "a"])],
    ["label", "raw"])
~~~